mc.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: 2018-06-01

```

Comment:
What is the pupose of this file? It's not referenced in the build pipeline and isn't a kubernetes file.

Reply:
Matthew mentioned that he had struggled to set up an AKS instance, so used some other Azure container deployment service to test out the web app and API together in the sandbox environment. I assume this file is the configuration for that, in which case I think it should come out of this PR.

Reply:
I think there's a rogue travis.yml file that can go too.

Reply:
This file was never supposed to be added. I did add the filename to .gitignore, but I seem to have lost that change somewhere.
This was used to run multi-containers on Azure for the show and tell, and can be removed without problems.